src/test/java/com/team4/testingsystem/services/impl/ChosenOptionImplTest.java

```diff
@@ -93,4 +91,17 @@ void saveFail(){
                 () -> chosenOptionService.save(chosenOption));
     }
 
+    @Test
+    void saveAllSuccess(){
+
+        List<ChosenOption> chosenOptions = new ArrayList<>();
+        chosenOptions.add(new ChosenOption());
+        chosenOptions.add(new ChosenOption());
+        chosenOptions.add(new ChosenOption());
```

Comment:
List<ChosenOption> chosenOptions = Lists.of(new ChosenOption(), new ChosenOption(), new ChosenOption())